docs/src/main/paradox/docs/delta/api/current/assets/views/elasticsearch/create-put.sh

```diff
@@ -0,0 +1,33 @@
+curl -XPUT \
+-H "Content-Type: application/json" \
+"http://localhost:8080/v1/views/myorg/myproj/myview" -d \
+'{
+    "@type": [
```

Comment:
No need for an array, `"@type": "ElasticSearchView"` would be more readable (same on create-post.sh)